domain.md

```diff
@@ -0,0 +1,71 @@
+In order to deploy CAM that is integrated with customer own Active Directory (AD), the folloing requirements must be met. When deploying CAM, the **account credential and DC fqdn** must be provided, and the DC must be able to communicated with the given **fqdn**.
+
+# 1) Creating an account with delegated permissions to join a computer to domain. 
+
+The following steps setup an account with delegated privileges to join a computer to domain, assuming an account already created, otherwise, please create an account first.
+
+ * Login Domain Controller Server.
+ * Lanuch Active Directory Users and Computers.
+ * Locate the **OU** that you want to join a computer to and right-click and then click Delegate Control. 
+ * The **Delegation of Control Wizard** opens, click **Next**
+ * In the **Users or Groups** page, click **Add** to add a specific user or a specific group to the **Selected users and groups** list, and then click **Next**
+ * In the **Tasks to Delegate** page, click **Create a custom task to delegate**, and then click **Next**.
+ * In the **Active Directory Object Type** page, Click **Only the following objects in the folder**, and then from the list, click to select the **Computer objects** check box. Then, select the check boxes below the list, **Create selected objects in this folder** and **Delete selected objects in this folder**, Click **Next**.
+ * In the **Permissions** page, click to select the following permission check boxes from list:  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Reset Password**  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Read and write Account Restrictions**  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Validated write to DNS host name**  
+       &nbsp;&nbsp;&nbsp;&nbsp;**Validated write to service principal name**
+ * Click **Next** to **the Completing the Delegation of Control Wizard** page, after review then click **Finish**.
+
+
+# 2) Enable remote powershell admin on a Domain Controller (DC) machine. 
+
+The following steps enable remote powershell admin
+
+* Login Domain Controller Server.
+* Start Windows PowerShell as an administrator.
+* Check the configuration of WinRM service by running following command:  
+  **Get-WmiObject Win32_Service | Where-Object { $_.Name -eq 'WinRM'} | Format-Table**  
+  The value of the StartMode property in the output should be "Auto" and the value of the Status property in the output should be "Running"
+* Set the StartMode property of WinRM service to Automatic by running following command if needed:  
+  **Set-Service WinRM -StartupType Automatic**  
```

Comment:
Online you see a bunch of people recommending winrm-quickconfig. But it seems to enable http by default (yech) - did you look at that?

Reply:
In azure VM, I use Enable-PSRemoting –force command. From my machine, I have to use https to connect, otherwise it failed.  If it is in the trusted domain, it might use http, it not, you must either use HTTPS as the transport or add the remote machine to the TrustedHosts configuration settings

Reply:
Okay. Yeah good to confirm that http doesn't work.

Reply:
Or maybe that's incorrect - if both client and server are part of the same (or trusted) domain then they might negotiate different transport security than https, using Kerberos or CredSSP or something. So it's more just knowing that we haven't enabled 'cleartext' transport. Not sure how to verify that. Like, why would winrm-quickconfig enable http? Seems ludicrous in today's day and age unless there's some other trick we're missing.